support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/stash/StashStep/help-useDefaultExcludes.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    If selected, use the default excludes from Ant - see <a href="http://stackoverflow.com/a/6351685">here</a> for the list.
```

Comment:
Perhaps better to link directly to the [manual](http://ant.apache.org/manual/dirtasks.html#defaultexcludes).

BTW I like to use `target="_blank"` in all links in help text, to work around [JENKINS-18435](https://issues.jenkins-ci.org/browse/JENKINS-18435).